webhook/validator/eventbus.go

```diff
@@ -64,7 +66,21 @@ func (eb *eventbus) ValidateUpdate(ctx context.Context) *admissionv1.AdmissionRe
 				return DeniedResponse("\"spec.nats.exotic.auth\" is immutable, can not be updated")
 			}
 		}
+	case eb.neweb.Spec.JetStream != nil:
+		if eb.oldeb.Spec.JetStream == nil {
+			return DeniedResponse("Can not change event bus implmementation")
```

Comment:
typo

Reply:
not seeing the typo - where? This is the case that somebody used to have a "jetstream" section and now doesn't, right? (thus changing the eventbus) This is the same as the "nats" check above on line 49. Admittedly, the error message could be more specific.

Reply:
`implmementation`

Reply:
thanks. will fix in both spots